packages/server/src/error/TRPCError.ts

```diff
@@ -20,7 +20,7 @@ export function getTRPCErrorFromUnknown(cause: unknown): TRPCError {
 }
 
 export class TRPCError extends Error {
-  public readonly cause?: Error;
+  public override readonly cause?: Error;
```

Comment:
What happens in older TS versions when the `override`-keyword is encountered?

Reply:
I can't reproduce it outside of the legacy project I'm working on, it's probably some kind of a weird combination of dependencies, can't update them all (especially transitive ones) in this repo. I'm going to go ahead and cancel this PR, sorry for disturbance.